test/eq-by-simplification.js

```diff
@@ -0,0 +1,34 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/eq-by-simplification';
+
+const ruleTester = avaRuleTester(test, {
+    env: {
+        es6: true
+    },
+    parserOptions: {
+        sourceType: 'module'
+    }
+});
+
+const error = {
+    ruleId: 'eq-by-simplification',
+    message: '`eqBy(prop(_))` should be simplified to `eqProps(_)`'
+};
+
+ruleTester.run('filter-simplification', rule, {
+    valid: [
+        'eqBy(trim, \'\')',
+        'R.eqBy(R.pluck(key))'
```

Comment:
What about `eqBy(prop('foo', bar))`? This might be valid and can’t be substituted with `eqProps`.